src/custom/components/swap/TradePrice/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useMemo } from 'react'
+import TradePriceMod, { TradePriceProps } from './TradePriceMod'
+import { useUSDCValue } from 'hooks/useUSDCPrice'
+import { formatSmart } from 'utils/format'
+import { tryParseAmount } from 'state/swap/hooks'
+import { SHORT_PRECISION } from '@src/custom/constants'
+
+export * from './TradePriceMod'
+
+export default function TradePrice(props: Omit<TradePriceProps, 'fiatValue'>) {
+  const { price, showInverted } = props
+
+  const priceSide = useMemo(
+    () =>
+      !showInverted
+        ? tryParseAmount(price.invert().toFixed(), price.baseCurrency)
+        : tryParseAmount(price.toFixed(), price.quoteCurrency),
+    [price, showInverted]
+  )
+  const amount = useUSDCValue(priceSide)
+  const fiatValueFormatted = formatSmart(amount, SHORT_PRECISION)
```

Comment:
SHORT_PRECISION === 4?
(See my other comment) Can we use 2 for fiat?

Reply:
yes!

Reply:
c200c53